webtop/__init__.py

```diff
@@ -133,63 +139,53 @@ def render_stats(stats: dict, _format: str) -> str:
         output = json.dumps(stats, indent=2)
     elif _format == "yaml":
         output = yaml.dump(stats, default_flow_style=False, sort_keys=False)  # type: ignore
+    # noinspection PyUnboundLocalVariable
     return output
 
 
-def main() -> None:
+async def main() -> None:
     args = parse_args()
     assert are_args_valid(args)
 
+    url: URL = args.url
```

Comment:
variable assignment here is redundant as `args.url` already has a clear name and correct type. Even if you wanted to rename the var, vars should be declared as close as possible to where they're used- and this assignment is floating way above where it's used later.

Reply:
Ahh, that's actually an artifact of the original Requests-based `--resolve` option. Probably not needed anymore since we're not going to be modifying the URL for that option.